### PR TITLE
Bug fix ROC-3232 to disable make an offer link when offer submitted

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-3232

### Change description

Bug fix to disable make an offer link and content if a defendant has already submitted an offer

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No

### CODE REVIEW

File: src/main/features/dashboard/views/defendant.njk [hunk 1]
```diff
@@ -22,15 +22,11 @@
             {{ t('Sign the agreement') }}
           </a>
         </p>
-      {% elif featureToggles.offer and not claim.settlementReachedAt  %}
+        {% elif featureToggles.offer and claim.status === ClaimStatus.CLAIM_REJECTED  %}
```
Comment (timja-kainos) [inline src/main/features/dashboard/views/defendant.njk]: won't this block new offers?

Comment (rattur) [reply src/main/features/dashboard/views/defendant.njk]: Will check that scenario

Comment (rattur) [reply src/main/features/dashboard/views/defendant.njk]: From my discussion content designer, a defendant currently can only make one offer. In the future, there probably will be some functionality to deal with multiple offers. Hope this helps Tim

Comment (geek-soft) [reply src/main/features/dashboard/views/defendant.njk]: I always thought that defendant can make as many offers as he would like as long as claimant rejected previous offer. In other words many concurrent offers should not be possible but many offers in single offer chain should be allowed.

Please check with business.

Comment (timja-kainos) [reply src/main/features/dashboard/views/defendant.njk]: on a closer look it looks right, i got mixed up with offer rejected.
but need to test with:
no offer made -> visible
offer made -> not visible
offer rejected -> visible
offer accepted -> not visible
countersigned -> not visible

Comment (timja-kainos) [reply src/main/features/dashboard/views/defendant.njk]: I tested this, It fixes the current bug,
I noticed another bug where rejecting an offer doesn't update what people can see.
i.e. claimant still gets 
> The defendant has made an offer to settle out of court.
View their offer 

defendant still get's
> You made an offer to settle out of court.

Then you get an error if you try reject again.
Hopefully its fixed in @varma02 and @geek-soft 's counterclaim PR

Comment (timja-kainos) [reply src/main/features/dashboard/views/defendant.njk]: @pragnesh-patel @vianvi FYI

Comment (geek-soft) [reply src/main/features/dashboard/views/defendant.njk]: Yeah, rejecting an offer is not handled at all. When I merged counted claim story I will have closed look to make sure all the statuses are right and I will fix few more bugs if there is not story already for it.

**Comment by geek-soft** [comment]
Closing as https://github.com/hmcts/cmc-citizen-frontend/pull/401 supersedes it.

**Pull request closed without merging**